docs/people-and-groups/authenticating-with-jwt.md

```diff
@@ -8,7 +8,7 @@ redirect_from:
 
 {% include plans-blockquote.html feature="JWT-based authentication" %}
 
-You can connect Metabase to your identity provider using JSON Web Tokens (JWT) to authenticate people.
+You can connect Metabase to your identity provider using JSON Web Tokens (JWT) to authenticate people. If the user doesn't exist, Metabase will create it on the fly.
```

Comment:
```suggestion
You can connect Metabase to your identity provider using JSON Web Tokens (JWT) to authenticate people. If the user doesn't exist, Metabase will create an account on the fly.

Metabase accounts created with an IdP login don't have passwords. People who sign up for Metabase using an IdP must continue to use the IdP to log into Metabase.
```

We might want to copy this note over to the SAML doc as well, if it is a common question (and if the note about "no passwords" applies to all accounts created with SSO).

https://www.metabase.com/docs/latest/people-and-groups/authenticating-with-saml